app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/AboutFragment.kt

```diff
@@ -142,6 +148,24 @@ class AboutFragment : Fragment() {
                 "https://github.com/splitwise/TokenAutoComplete/",
                 "Apache License, Version 2.0",
             ),
+            Library("accompanist","https://github.com/google/accompanist","Apache License, Version 2.0"),
+            Library("FlexboxLayout","https://github.com/google/flexbox-layout","Apache License, Version 2.0"),
+            Library("Error Prone","https://github.com/google/error-prone","Apache License, Version 2.0"),
+            Library("ListenableFuture",
+                "https://mvnrepository.com/artifact/com.google.guava/listenablefuture/1.0",
+                "Apache License, Version 2.0"
+            ),
+            Library("ColorPicker","https://github.com/gregkorossy/ColorPicker","Apache License, Version 2.0"),
+            Library("DateTimePicker","https://github.com/gregkorossy/DateTimePicker","Apache License, Version 2.0"),
+            Library("jcip-annotations","https://jcip.net/","Public License"),
+            Library("Http Components","https://hc.apache.org/","Apache License, Version 2.0"),
+            Library("Compose Runtime",
```

Comment:
Please change the name to "Jetbrains Compose Runtime" and use the URL https://github.com/JetBrains/compose-multiplatform-core